examples/games/breakout.rs

```diff
@@ -270,7 +270,8 @@ fn setup(
 
     // Because we need to round the number of columns,
     // the space on the top and sides of the bricks only captures a lower bound, not an exact value
-    let center_of_bricks = (LEFT_WALL + RIGHT_WALL) / 2.0;
+    //
+    let center_of_bricks = 0.0;
```

Comment:
This only works when LEFT_WALL + RIGHT_WALL == 0. Part of the appeal of using constants like this is that readers can twiddle with them freely without breaking the game.